### PR TITLE
Feat/install xyz tech

### PR DESCRIPTION
**Pull request by Atishaya7777**

### CODE REVIEW

File: Test.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class Test {
+    public static void main(String[] args) {
+        System.out.println("Hello world!");
```
Comment (Atishaya7777) [inline Test.java:3]: Yo, this is wrong. You should have done this instead.

**Review by Atishaya7777 — COMMENTED** [review]
Review completed.

**Comment by samuel-ah** [comment]
looks ok put in prod asap

File: Test.java (hunk 1 above)
Comment (madhurSingla) [reply Test.java:3]: nah, go ahead and kill Kha

**Pull request merged**